### PR TITLE
ENH: projects: add up-for-grabs

### PR DESCRIPTION
**Pull request by SantiagoTorres**
Add an entry to the [in-toto project](https://in-toto.io).

### CODE REVIEW

**Comment by SantiagoTorres** [comment]
Hi again!

I finally found a pocket of time to add one of the projects I'm a contributor for. I'll be assigning more issues with the up for grabs tag sometime soon. Hopefully everything is in order in this PR! :)

File: _data/projects/in-toto.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: in-toto
+desc: A software supply chain security framework
+site: https://in-toto.io/
+tags:
+- Python
+- Security
+- Software Supply Chain
+- CI/CD
+upforgrabs:
+  name: up for grabs
```
Comment (ritwik12) [inline _data/projects/in-toto.yml]: @SantiagoTorres It's "Up for grabs" according to your label. Please correct it.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/in-toto.yml (hunk 1 above)
Comment (SantiagoTorres) [reply _data/projects/in-toto.yml]: Oh woops! Thanks for the review!

**Comment by SantiagoTorres** [comment]
Let me know if this is ok now :)

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@SantiagoTorres Thanks!! and sorry for the delay. I was a bit busy.

**Comment by SantiagoTorres** [comment]
@ritwik12 No, thanks to you for the review and merge! :+1:

**Pull request merged**